enarx-keep/build.rs

```diff
@@ -59,22 +59,42 @@ fn main() {
         })
         .collect();
 
-    for entry in std::fs::read_dir("shims").unwrap() {
+    // internal crates are not included, if there is a `Cargo.toml` file
+    // trick cargo by renaming the `Cargo.toml` to `Cargo.tml` before
+    // publishing and rename it back here.
+    for entry in std::fs::read_dir("internal").unwrap() {
+        let shim_path = entry.unwrap().path();
+
+        let cargo_toml = shim_path.join("Cargo.toml");
+        let cargo_tml = shim_path.join("Cargo.tml");
+
+        if cargo_tml.exists() {
+            std::fs::copy(cargo_tml, cargo_toml).unwrap();
+        }
+    }
```

Comment:
It is my understanding that this modifies files in the source directory. I think that's frowned upon. But... they can continue frowning until `cargo` provides us a better solution.

Reply:
yep